tests/basic_test.go

```diff
@@ -40,13 +40,8 @@ func TestBasicPolicy(t *testing.T) {
 
 func testEmptyPolicy(t *testing.T) {
 	t.Run("is empty", func(t *testing.T) {
-		s, err := nmpolicy.GenerateState(types.PolicySpec{}, nil, types.NoCache())
-
-		assert.NoError(t, err)
-
-		expectedEmptyState := types.GeneratedState{MetaInfo: types.MetaInfo{Version: "0"}}
-		assert.NotEqual(t, time.Time{}, s.MetaInfo.TimeStamp)
-		assert.Equal(t, expectedEmptyState, resetTimeStamp(s))
+		_, err := nmpolicy.GenerateState(types.PolicySpec{}, nil, types.NoCache())
```

Comment:
Please add a test for the commit scenario.

Reply:
The `testPolicyWithFilterCaptureAndDesiredStateCaptureEntryRef` is the one doing it.